core/sql/regress/executor/EXPECTED130

```diff
@@ -124,7 +134,16 @@ inserted row13
 
 --- 3 row(s) selected.
 >>
->>
+>>values (filetolob('./myfile.txt'));
+
+*** ERROR[8442] Unable to access ExpLOB interface. Call to ExpLOB returned error LOB_PTR_ERROR(532). Error detail 0.
```

Comment:
I was thinking about that initially but had to fix the invalid lobHandle issue anyway. We do have checks in several functions in the binder to prevent LOB functions but not sure if it's comprehensive. Let me see if it'll be easy to detect this within a values clause. It will be an additional check at compile time  and the invalid lobHandle check at runtime  will still remain.